### PR TITLE
Add fluent_bit_vendor in humble

### PR DESCRIPTION
**Pull request by Minipada**
Please add the following dependency to the rosdep database.

## Package name:

fluent_bit_vendor

## Package Upstream Source:

https://github.com/Minipada/fluent_bit_vendor

## Purpose of using this:

I want to use Fluent Bit as a third party library for https://github.com/Minipada/ros2_data_collection
At the moment it is in this repo but I want to take it out.

Distro packaging links:

## Links to Distribution Packages

# Please Add This Package to be indexed in the rosdistro.

humble

# The source is here:

https://github.com/fluent/fluent-bit

# Checks
 - [X] All packages have a declared license in the package.xml
 - [X] This repository has a LICENSE file
 - [X] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Comment by Minipada** [comment]
I am not sure what is wrong here. I can fix it but I would need a hint.

**Comment by Minipada** [comment]
I can't get the problem:
`FAILED test/test_build_caches.py::test_build_caches - RuntimeError: humble: missing package.xml file for package "fluent_bit_vendor"

It is here https://github.com/Minipada/fluent_bit_vendor/blob/humble/package.xml#L4

File: humble/distribution.yaml [hunk 1]
```diff
@@ -1359,6 +1359,16 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: humble-devel
     status: maintained
+  fluent_bit_vendor:
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/minipada/fluent_bit_vendor-release.git
```
Comment (quarkytale) [inline humble/distribution.yaml:1366]: My initial guess is a problem with the release repository it doesn't have the release/humble/{package}/{version} branch. I see you are trying to create a release in ros2-gbp, so first just add the source entry here, then continue the process in ros2-gbp, then you can create another PR to add the release repository. More details here https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html#what-if-my-existing-release-repo-isn-t-on-ros2-gbp.
Also please consider following the recommended release workflow using Bloom https://docs.ros.org/en/rolling/How-To-Guides/Releasing/First-Time-Release.html.

**Review by quarkytale — CHANGES_REQUESTED** [review]

**Comment by Minipada** [comment]
Thanks for the links... I tried to follow the steps and even asked on the discord.
Seems ok now

**Review by quarkytale — APPROVED** [review]

**Pull request merged**